src/module-2/components/stylesheets/index.css

```diff
@@ -0,0 +1,42 @@
+*{
```

Comment:
please use some other name for this file, for example, base.css, or create 2 separate css files: `global.css` with *? body and tagname styles + overrides, and `layout.css` with `.results` and `.filters`

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)